app/src/main/java/com/davidtomas/taskyapp/features/agenda/data/event/local/source/EventLocalSourceImpl.kt

```diff
@@ -19,6 +19,10 @@ class EventLocalSourceImpl(
         }
     }
 
+    override suspend fun saveEvents(events: List<EventModel>) {
+        events.forEach { saveEvent(it) }
+    }
```

Comment:
Might want to expose a DB function for saving a list of events, otherwise you also insert them all sequentially here